README.md

```diff
@@ -1,74 +1,100 @@
-    | Topic Tag       | Usage                                    |
-    | --------------- | ---------------------------------------- |
-    | az              | For actions related to Azure             |
-    | code            | For actions related to building code     |
-    | certs           | For actions related to certificates      |
-    | db              | For actions related to databases         |
-    | git             | For actions related to Git               |
-    | iis             | For actions related to IIS               |
-    | microsoft-teams | For actions related to Microsoft Teams   |
-    | svc             | For actions related to Windows Services  |
-    | jira            | For actions related to Jira              |
-    | meta            | For actions related to running workflows |
-    | pagerduty       | For actions related to PagerDuty         |
-    | test            | For actions related to testing           |
-    | tf              | For actions related to Terraform         |
-  - [ ] Add any additional topics for an action if they apply    
-    
+# App Pool Status
+
+This action will return the status of an on-premises IIS app pool
+
+## Index <!-- omit in toc -->
+
+- [Inputs](#inputs)
+- [Outputs](#outputs)
+- [Prerequisites](#prerequisites)
+- [Example](#example)
+- [Code of Conduct](#code-of-conduct)
+- [License](#license)
 
 ## Inputs
-| Parameter | Is Required | Description           |
-| --------- | ----------- | --------------------- |
-| `input-1` | true        | Description goes here |
-| `input-2` | false       | Description goes here |
+
+| Parameter                  | Is Required | Description                          |
+| -------------------------- | ----------- | ------------------------------------ |
+| `server`                   | true        | The name of the target server        |
+| `app-pool-name`            | true        | IIS app pool name                    |
+| `service-account-id`       | true        | The service account name             |
+| `service-account-password` | true        | The service account password         |
+| `server-public-key`        | true        | Path to remote server public ssl key |
+
 
 ## Outputs
-| Output     | Description           |
-| ---------- | --------------------- |
-| `output-1` | Description goes here |
+
+| Parameter         | Description                    |
+| ----------------- | ------------------------------ |
+| `app-pool-status` | The status of the IIS app pool |
+
+## Prerequisites
+
+The IIS app pool status action uses Web Services for Management, [WSMan], and Windows Remote Management, [WinRM], to create remote administrative sessions. Because of this, Windows OS GitHubs Actions Runners, `runs-on: [windows-2019]`, must be used. If the IIS server target is on a local network that is not publicly available, then specialized self hosted runners, `runs-on: [self-hosted, windows-2019]`,  will need to be used to broker commands to the server.
```

Comment:
```suggestion
The IIS app pool status action uses Web Services for Management, [WSMan], and Windows Remote Management, [WinRM], to create remote administrative sessions. Because of this, Windows Actions Runners, `runs-on: [windows-2019]`, must be used. If the IIS server target is on a local network that is not publicly available, then specialized self-hosted runners, `runs-on: [self-hosted, windows-2019]`,  will need to be used to broker commands to the server.
```